resources/puppetlabs/lein-ezbake/template/global/ext/fpm.rb

```diff
@@ -4,13 +4,22 @@
 require 'optparse'
 require 'ostruct'
 
+# ezbake.rb is rendered from
+# resources/puppetlabs/lein-ezbake/staging-templates/ezbake.rb.mustache
+#
+# inside the build container, fpm.rb is at:
+# /code/target/staging/puppetserver-8.9.0/ext/fpm.rb
+# ezbake:
+# /code/target/staging/ezbake.rb
+require_relative '../../ezbake.rb'
```

Comment:
I really appreciate the comment. It makes me think it's not present in the built tarball. Is that a problem?

How about:
```ruby
begin
  require_relative '../../ezbake.rb'
rescue LoadError
  options.java_bin = '/usr/bin/java'
else
  options.java_bin = EZBake::Config[:java_bin]
end
```

Or some variant of that.